From fa7a257113e2cfc963a0be9dca5d7b4c73999dcc Mon Sep 17 00:00:00 2001
From: Havard Graff <havard.graff@gmail.com>
Date: Tue, 12 May 2015 10:45:13 +0200
Subject: [PATCH] Add configure-option to disable building docs

---
 Makefile.am  | 6 +++++-
 configure.ac | 7 +++++++
 2 files changed, 12 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6fb3d473..5712edd8 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -15,7 +15,11 @@ EXTRA_DIST = LICENSE ChangeLog.v1 ChangeLog.libgcj			\
 	libffi.xcodeproj/project.pbxproj				\
 	libtool-ldflags
 
-info_TEXINFOS = doc/libffi.texi
+
+info_TEXINFOS =
+if BUILD_DOCS
+#info_TEXINFOS += doc/libffi.texi
+endif
 
 ## ################################################################
 
diff --git a/configure.ac b/configure.ac
index 445c589e..669c9f5b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -282,6 +282,13 @@ if test "x$GCC" = "xyes"; then
   fi
 fi
 
+AC_ARG_ENABLE(docs,
+              AC_HELP_STRING([--disable-docs],
+                             [Disable building of docs (default: no)]),
+              [enable_docs=no],
+              [enable_docs=yes])
+AM_CONDITIONAL(BUILD_DOCS, [test x$enable_docs = xyes])
+
 AH_BOTTOM([
 #ifdef HAVE_HIDDEN_VISIBILITY_ATTRIBUTE
 #ifdef LIBFFI_ASM
